README.md

```diff
@@ -106,8 +106,74 @@ or in your Visual Studio Package.appxmanifest capabilities.
 
 ---
 
+## SpatialMappingComponent
```

Comment:
Could you also please add a test in Tests folder? That really helps with avoiding regressions when others change code. Also a good way to show usage.